<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dockWidget_trends_earth</class>
 <widget class="QDockWidget" name="dockWidget_trends_earth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trends.Earth</string>
  </property>
  <widget class="QWidget" name="dock_widget_vlayout">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_algorithms">
       <attribute name="title">
        <string>Algorithms</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTreeView" name="algorithms_tv">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectColumns</enum>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_datasets">
       <attribute name="title">
        <string>Datasets</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_refresh">
            <property name="toolTip">
             <string>Refresh dataset list manually.
This scans the Trends.Earth base directory for new datasets.
It also scans the remote Trends.Earth server 
for running algorithms and their results.</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QgsFilterLineEdit" name="lineEdit_search">
            <property name="toolTip">
             <string>Filter the list of datasets according with their name</string>
            </property>
            <property name="placeholderText">
             <string>Search...</string>
            </property>
            <property name="showSearchIcon">
             <bool>true</bool>
            </property>
            <property name="qgisRelation" stdset="0">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_filter">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QgsCollapsibleGroupBox" name="date_filter_group">
          <property name="toolTip">
           <string/>
          </property>
          <property name="title">
           <string>Filter by date</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="collapsed">
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>End</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QgsDateTimeEdit" name="start_dte">
             <property name="toolTip">
              <string/>
             </property>
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2021</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QgsDateTimeEdit" name="end_dte">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2021</year>
               <month>12</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="date">
              <date>
               <year>2021</year>
               <month>12</month>
               <day>1</day>
              </date>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="datasets_tv">
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="import_dataset_pb">
            <property name="text">
             <string>Import dataset...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="create_layer_pb">
            <property name="text">
             <string>Create...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_download">
            <property name="toolTip">
             <string>Download datasets from the official Trends.Earth remote server</string>
            </property>
            <property name="text">
             <string>Download data...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_load">
            <property name="toolTip">
             <string>Load a set of base layers that were prepared by Trends.Earth into QGIS in order to serve as a reference map</string>
            </property>
            <property name="text">
             <string>Load Base Map</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>359</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>qgsdatetimeedit.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgsfilterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
